core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -65,7 +70,7 @@ var ExplorationEditorPage = function() {
   var sharePublishModalElement = element(
     by.css('.e2e-test-share-publish-modal'));
   var selectionRenderedElement = element(
```

Comment:
Can we change the variable name to better reflect the `.e2e-test--exploration-category-dropdown`?

Reply:
done